src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,51 @@
+<div class="container" fxFlex="50%">
+  <mat-card>
+    <form [formGroup]="officeForm" (ngSubmit)="submit()">
+      <mat-card-content>
+        <div fxLayout="column" fxLayoutAlign="center">
+
+          <mat-form-field fxFlex="100%">
```

Comment:
fxFlex is also not required here as the fxLayout is column. Please remove from all form fields.